### DISCUSSION TITLE
How do you add headers to HTTP requests?

### DISCUSSION THREAD

**Discussion by YohannesTz** [Sketchware-Pro/Sketchware-Pro#851] [Q&A]
How do you add headers to HTTP requests? Any link to an example or documentation would be helpful.

I am trying to attach Authentication Barear to a post request.

**Comment by nicesapien-too** [comment]
Hey there! I’m the creator of the swpro docs. I will add a doc on that in 3 days. If your issue is already resolved, please let me know. Thanks!
